### ISSUE TITLE
Simulate a UITableViewCell selection, tests fail

### ISSUE THREAD

**Issue by coulton** [kif-framework/KIF#489]
I'm attempting to simulate the selection of a UITableViewCell. The tests pass if I use the deprecated method `tapRowInTableViewWithAccessibilityLabel:atIndexPath:`. However, it fails if I use the recommended `tapRowAtIndexPath:inTableViewWithAccessibilityIdentifier:`.

```
[self.tableView setAccessibilityLabel:@"ListTableView"];

NSIndexPath *indexPath = ...
[tester tapRowAtIndexPath:indexPath inTableViewWithAccessibilityIdentifier:@"ListTableView"];
```

And the error that I get:

`test failure: -[TestLogin XXX] failed: The step timed out after 10.00 seconds: Could not find view matching: accessibilityIdentifier == "ListTableView"`

**Comment by Kowaio** [comment]
Hello coultonvento,

it seems that I have the same issue. Did you find why this is happening using the new method but works with the deprecated one?

Thanks a lot!

Regards,

**Comment by Kowaio** [comment]
Oh silly me. An identifier is needed, not a label. My bad.

Best,
